specification/hugr.md

```diff
@@ -243,7 +251,7 @@ regions:
     `AliasDef`.
 
 
-A **loadable HUGR** is a module HUGR where all edges are connected and there are
+A **loadable HUGR** is a module HUGR where all ports are connected and there are
```

Comment:
This is not quite correct anymore. The multiports (state edge ports, output classical value ports) can remain disconnected.

We still have to specify which ports can have multiple/no connection, so that can come in another PR.

Reply:
OK yes, we should at least allow discarded classical values to be unconnected ports, for self-consistency. Perhaps "all output ports are connected" will suffice?

Reply:
"all input ports" except input state edges

Reply:
Sorry I missed this comment before merging. Do state (order) edges have ports (at the spec level I mean)?

Reply:
Good question. I guess we should define how to talk about the "other edge" ports. (see #159)